src/components/Upload/Forms/DynamicForm.js

```diff
@@ -139,28 +165,35 @@ class DynamicForm extends Component {
 		}
 		
 		return (
-			<section id="dynamicUploadForm" className="container justify-content-center pt-4">
-				{this.renderSection(this.props.formDTD.standardFields, this.props.form, this.props.userInformation)}
-				{dynamicSections}
-                <Row className="dropzone btn-sm">
-                    <Col md={12}>
-                        <FileDropzone uploader={uploader} isUploading={this.props.isUploading}/>
-                    </Col>
-                </Row>
-        		<Row className="fixed-bottom pt-4" id="form-footer">
-        			<div className="container justify-content-center">
-	        			<Row className="text-center">
-		                    <Col md={12}>
-		                        <Button id="cancel" className="mr-3">Cancel</Button>
-		                        <Button id="submit" disabled={this.isSubmitDisabled()} type="primary" onClick={this.handleSubmit}>Upload</Button>
-		                    </Col>
-	                    </Row>
-                    </div>
-                </Row>
-			</section>
+			<React.Fragment>
+				<Prompt
+					when={() => {return true;}}
+					message={'Your data will be lost.  Press OK to continue or Cancel to stay.'}
+				/>
+				<article id="dynamicUploadForm" className="container justify-content-center pt-4">
+					{this.renderSection(this.props.formDTD.standardFields, this.props.form, this.props.userInformation)}
+					{dynamicSections}
+					<Row className="dropzone btn-sm">
+						<Col md={12}>
+							<FileDropzone uploader={uploader} isUploading={this.props.isUploading}/>
```

Comment:
Looks like we are still using isUploading here.

Reply:
We shouldn't be - thanks for that catch.

Reply:
Wait - there shouldn't be an issue with isUploading.  I only removed isQuerying.